CMakeLists.txt

```diff
@@ -1100,7 +1100,7 @@ if(NOT APPLE AND NOT ANDROID)
                     FILES
                     ${CMAKE_SOURCE_DIR}/default/data/art/icons/FO_Icon_${SIZE}x${SIZE}.png
                     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/icons/hicolor/${SIZE}x${SIZE}/apps/
-                    RENAME freeorion.png
+                    RENAME org.freeorion.FreeOrion.png
```

Comment:
@o01eg do you know if the icon files are used in any other context for which this would be problematic?

Reply:
I don't see if it used somewhere else so it should be fine. If something will be wrong it could be renamed also to follow guidelines.

Also, as I understand, it also happened in `release-v0.5` branch so maybe it should be backported to.